server.go

```diff
@@ -3955,6 +3953,18 @@ func (s *Server) runContinuousQuery(cq *ContinuousQuery) {
 	cq.mu.Lock()
 	defer cq.mu.Unlock()
 
+	// Get server CQ parameters under lock, to avoid races.
+	s.mu.RLock()
```

Comment:
Perhaps not. This may be rather pedantic. This was a drive-by change -- the real race is around `lastRun` which was being written and read from different goroutines.